SS2_SingleSample/STARPipeline_KCO.wdl

```diff
@@ -0,0 +1,174 @@
+task StarRun {
+  File input_fastq_read1
+  File input_fastq_read2
+  File gtf
+  File  starGenome
+
+  command {
+   tar -zxvf ${starGenome}
```

Comment:
If it's not that many files, like say 10 or less, it's probably better to upload them individually to the bucket and make each an input to this task. You can mv them to whatever directory you want in this command before running STAR. It avoids having to untar every time and makes it a bit clearer what files are being used.